module/stats.py

```diff
@@ -0,0 +1,22 @@
+"""/stats command"""
+from telegram import Update
+from telegram.ext import CallbackContext
+from module.shared import SELECT, STATS_MESSAGE
+from module.create_connection import connect_and_execute
+
+def stats(update: Update, context: CallbackContext) -> None:
+    """called by /stats
+        It returns the number of books that are now on sale
+
+        Args:
+            update: update event
+            context: context passed by the handler"""
+
+    query: str = "SELECT COUNT(*) FROM Market"
```

Comment:
`: str` this type is redundant, because the type in this case can be inferred

```suggestion
    query = "SELECT COUNT(*) FROM Market"
```